packages/jest-jasmine2/src/jasmine/Spec.js

```diff
@@ -121,10 +127,26 @@ Spec.prototype.onException = function onException(e) {
     return;
   }
 
+  const {expected, actual} = e;
+  let message;
+  if (expected && actual) {
+    const diffString = diff(expected, actual, {
+      expand: this.expand,
+    });
+    message = matcherHint('.toBe') +
+      '\n\n' +
+      `Expected value to be (using ===):\n` +
+      `  ${printExpected(expected)}\n` +
+      `Received:\n` +
+      `  ${printReceived(actual)}` +
+      (diffString ? `\n\nDifference:\n\n${diffString}` : '');
+  }
+
   this.addExpectationResult(
     false,
     {
-      matcherName: '',
+      matcherName: 'blah blah',
+      message,
```

Comment:
I've verified that adding this property (even when it's `undefined` in most cases) does not cause a negative impact on the other tests. Though the tests _are_ failing due to snapshot issues noted in #3216